libraries/typings-generator/src/TypingsGenerator.ts

```diff
@@ -46,12 +48,41 @@ export interface ITypingsGeneratorOptions<TTypingsResult = string | undefined>
   filesToIgnore?: string[];
 }
 
+/**
+ * @public
+ */
+export interface IReadFile<TFileContents = string> {
+  (filePath: string, relativePath: string): Promise<TFileContents> | TFileContents;
+}
```

Comment:
```suggestion
export type ReadFile<TFileContents = string> = (filePath: string, relativePath: string): Promise<TFileContents> | TFileContents;
```

Reply:
I think of types that are defined as interfaces (and start with `I`) as objects.